services/crud.py

```diff
@@ -95,5 +95,11 @@ def remove_user(db: Session, user_id: int):
         raise HTTPException(status_code=500, detail="An error occurred while deleting user")
 
```

Comment:
you are missing a repository layer where we communicate with the db, not in the service layer